hightouch_blueprints/verify_sync.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+import errors
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--sync-id', dest='sync_id', required=True)
+    parser.add_argument('--sync-run-id', dest='sync_run_id', required=False)
+    args = parser.parse_args()
+    return args
+
+
+
+def get_sync_status(sync_id, sync_run_id, access_token):
+    """
+    Gets the sync status from the Hightouch API
+    see: https://hightouch.io/docs/api-reference/#operation/ListSyncRuns
+    """
+    sync_api = f"https://api.hightouch.io/api/v1/syncs/{sync_id}/runs"
+    api_headers = {
+        'authorization': f"Bearer {access_token}",
+        'Content-Type': 'application/json'
+    }
+    params = {
+        "runId": sync_run_id
+    }
+    try:
+        sync_run_response = requests.get(sync_api,
+                                              params=params,
+                                              headers=api_headers)
+        
+        sync_status_code = sync_run_response.status_code
+        sync_run_json = sync_run_response.json()
+        # check if successful, if not return error message
+        if sync_status_code == requests.codes.ok:
+            return sync_run_json
+            
+        elif sync_status_code == 400: # Bad request
+            print("Sync status request failed due to Bad Request Error.")
+            sys.exit(errors.EXIT_CODE_BAD_REQUEST)
+            
+        elif sync_status_code == 401: # Incorrect credentials 
+            print("Incorrect credentials, check your access token")
+            sys.exit(errors.EXIT_CODE_INVALID_CREDENTIALS)
```

Comment:
Need a handling for wrong sync id. But this apparently just returns a successful response but with `{'data': []}`.